com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PrepareAppsDeploymentStep.java

```diff
@@ -50,6 +55,13 @@ protected StepPhase executeStep(ExecutionWrapper execution) {
         execution.getContext()
             .setVariable(Constants.EXECUTE_ONE_OFF_TASKS, true);
 

```

Comment:
Better way in my opinion is to delete if statement and assign direct  this statement 
 ProcessType.BLUE_GREEN_DEPLOY.equals(processType) to skipUpdateConfigurationEntries variable.
Make your choice :)

Reply:
Agree.